src/components/inline-link/docs.mdx

```diff
@@ -0,0 +1,93 @@
+---
+componentName: InlineLink
+---
+
+The `InlineLink` component should be used with a body of text. See more documentation [the HDS Product Figma](https://www.figma.com/file/noyY6dUMDYjmySpHcMjhkN/?node-id=2365%3A21590).
+
+Internally, this component uses two other components:
+
+- `MaybeInternalLink`: to automatically determine whether or not `next/link` should be used
```

Comment:
💭 : Based on some recent work by @dstaley, this `MaybeInternalLink` component might not be entirely necessary as `next/link` can handle external links without issue.

Reply:
Oooh interesting! Are you okay with a follow-up PR to handle experimenting with that & potentially removing `MaybeInternalLink`? It's used in quite a few places, might be easier to clean all of them up together?

Reply:
Works for me 👍